tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -271,11 +274,28 @@ def _check_osu_benchmarks_results(test_datadir, instance, mpi_version, benchmark
                 f"tolerated: {tolerated_value}, current: {value}"
             )
 
+            dimensions = {
```

Comment:
It may be useful to have at least some version numbers related to the EFA stack.

Also, for development purposes how difficult would it be to add the exact commits for CLI, cookbook and node?

Reply:
I agree with you. Due to the limited time, I will merge the PR and leave your suggestion for further improvement